cmd/ferretdb/main.go

```diff
@@ -69,6 +76,12 @@ var cli struct {
 
 	Test struct {
 		RecordsDir string `default:""  help:"Testing flag: directory for record files."`
+		Telemetry  struct {
+			URL            string        `default:"https://beacon.ferretdb.io/" help:"Testing flag: telemetry URL."`
```

Comment:
How would it work when we run FerretDB locally for dev/testing purposes? It seems that by default all the telemetry is sent to `beacon.ferretdb.io`. Is it on purpose? Will we be able to sort out dev instances etc?

Reply:
I don't think that really matters. We will be able to exclude our own instances just like we would exclude other testing instances – by small uptime.